### PR TITLE
Format 2101 User Guide

### PR DESCRIPTION
**Pull request by andrechuakj**
What have you done in this PR?
* Formatted 2101 user guide

To review:
* Grammar
* Printing
* Formatting

### CODE REVIEW

File: docs/2101UserGuide.md [hunk 1]
```diff
@@ -39,34 +46,33 @@ This guide uses the following features to make it easier for you to navigate aro
 :bulb: Boxes with the :bulb: icon contain additional tips and tricks to help you get the most out of CCACommander.
 </div>
 
-
-{:toc}
-
 --------------------------------------------------------------------------------------------------------------------
-
 ## Quick start
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
## Quick Start
```

File: docs/2101UserGuide.md [hunk 2]
```diff
@@ -265,50 +282,55 @@ Here’s how you can use it:<br>
...
 
 Here is an example command where we enrol the member at **member index** 1 to the event at **event index** 1, where our member
-has contributed 2 **hours**, and we want to leave a "Role: Photographer" **remark** to reflect their role as a photographer:<br>
+has contributed 2 **hours**, and we want to leave a "Role: Photographer" **remark** to reflect their role as a photographer:
+
 `enrol m/1 e/1 h/2 r/Role: Photographer`
 
 <hr class="command-separator">
 
 #### Edit an enrolment: `editEnrolment`
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
#### Edit an Enrolment: `editEnrolment`
```

File: docs/2101UserGuide.md [hunk 3]
```diff
@@ -516,6 +545,7 @@ This glossary provides definitions for the special terms used in this user guide
 
 --------------------------------------------------------------------------------------------------------------------
 
+<div style="page-break-after: always;"></div>
 ## Command summary
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
## Command Summary
```

File: docs/2101UserGuide.md [hunk 4]
```diff
@@ -532,14 +562,15 @@ Action | Format, Examples
...
+**Find member in list** | `findMember MEMBER_NAME [MORE_NAMES]...` <br> e.g.`findMember alice`
+**Find event in list** | `findEvent EVENT_NAME [MORE_NAMES]...` <br> e.g.`findEvent party`
 **List all members and all events** | `list`
 **Redo** | `redo`
 **Undo** | `undo`
 **Help** | `help`
 **Exit** | `exit`
 
+<div style="page-break-after: always;"></div>
 ## List of acceptable values
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
## List of Acceptable Values
```

File: docs/2101UserGuide.md [hunk 5]
```diff
@@ -1,15 +1,22 @@
...
+<div style="page-break-after: always;"></div>
+
+## Table of Contents
+{: .no_toc }
+1. Table of Contents
+{:toc}
+
+<div style="page-break-after: always;"></div>
 ## Introduction
 As a Co-curricular Activity (CCA) head, do you find yourself struggling to manage your CCA members’ attendance and record the array of events that your CCA has?
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
As a Co-curricular Activity (CCA) head, do you find yourself struggling to manage your CCA members’ attendances and record the array of events that your CCA has?
```

File: docs/2101UserGuide.md [hunk 6]
```diff
@@ -1,15 +1,22 @@
...
 Are the heaps of spreadsheets you are using to manage them making you feel overwhelmed? Well, fear not, **CCACommander Ultra ProMax Xtra 9000PLUS** is here to save the day!
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
Are the heaps of spreadsheets you are using to manage them making you feel overwhelmed? Well, fear not, **CCACommander Ultra Promax Xtra 9000PLUS** is here to save the day!
```

File: docs/2101UserGuide.md [hunk 7]
```diff
@@ -39,34 +46,33 @@ This guide uses the following features to make it easier for you to navigate aro
...
 
-
-{:toc}
-
 --------------------------------------------------------------------------------------------------------------------
-
 ## Quick start
 <div markdown="block" class="alert alert-warning">
 
 :warning: We assume that you have some basic knowledge of terminal commands. You can learn how to use the terminal with [this guide](https://www.freecodecamp.org/news/command-line-for-beginners/)
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
:warning: We assume that you have some basic knowledge of terminal commands. You can learn how to use the terminal with [this guide](https://www.freecodecamp.org/news/command-line-for-beginners/).
```

File: docs/2101UserGuide.md [hunk 8]
```diff
@@ -39,34 +46,33 @@ This guide uses the following features to make it easier for you to navigate aro
...
 
 </div>
 
-1. Ensure you have Java `11` or above installed in your computer by following [this guide](https://www.freecodecamp.org/news/command-line-for-beginners/).
+1. Ensure you have Java `11` or above installed in your computer by following [this guide](https://phoenixnap.com/kb/check-java-version-on-mac-windows).
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
1. Ensure you have Java `11` installed in your computer by following [this guide](https://phoenixnap.com/kb/check-java-version-on-mac-windows).
```

File: docs/2101UserGuide.md [hunk 9]
```diff
@@ -39,34 +46,33 @@ This guide uses the following features to make it easier for you to navigate aro
...
 3. Put the downloaded file into a folder of your choice (preferably a new empty folder).
 
-4. Open a command terminal in the folder you put the CCACommander file in, and use the `java -jar ccacommander.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+4. Open a command terminal in the folder you put the CCACommander file in, and use the `java -jar ccacommander.jar` command to run the application. A GUI resembling the one shown below should appear in a few seconds. Note how the app contains some sample data.<br>
+    <figure>
+        <img src="images/Ui.png"
+             alt="Ui">
+        <figcaption>Fig 1: Application upon running</figcaption>
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]: Should the figcaption be changed to something more descriptive like "Initial application display".

File: docs/2101UserGuide.md [hunk 10]
```diff
@@ -100,58 +110,59 @@ Light | Dark
 **Event List**: View the events stored in CCACommander. <br>
 **Storage Location**: Locate where your CCACommander data is stored in your computer.
 
-### Command History
+## Command History
 You can quickly recall previously entered commands for convenience as CCACommander stores almost all (see the info card below to see what commands are stored) commands entered by the user. <br/>
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
You can quickly recall previously entered commands for convenience as CCACommander stores almost all (see the info card below to see what commands are stored) commands entered by the user. 
<br>
```

Currently, in the PDF format, there is no spacing between this sentence and the next sentence.
Screenshot 2023-11-13 at 11 42 52 PM

File: docs/2101UserGuide.md [hunk 11]
```diff
@@ -100,58 +110,59 @@ Light | Dark
...
 Upon selecting the command box, you can use the <kbd>↑</kbd> to <kbd>↓</kbd> key to navigate through the commands entered .
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
Upon selecting the command box, you can use the <kbd>↑</kbd> to <kbd>↓</kbd> key to navigate through the commands entered.
```

File: docs/2101UserGuide.md [hunk 12]
```diff
@@ -100,58 +110,59 @@ Light | Dark
...
 
 Example:
-1. You have just marked attendance for the first member in the member list using the editEnrolment m/1 e/1 r/Present command.
-2. You can then use <kbd>↑</kbd> to get editEnrolment m/1 e/1 r/Present
-3. You can edit the member index to get editEnrolment m/2 e/1 r/Present to mark the second member as present.
+1. You have just marked attendance for the first member in the member list using the `editEnrolment m/1 e/1 r/Present` command.
+2. You can then use <kbd>↑</kbd> to get `editEnrolment m/1 e/1 r/Present`
+3. You can edit the member index to get `editEnrolment m/2 e/1 r/Present` to mark the second member as present.
 
 <div markdown="block" class="alert alert-info">:information_source: CCACommander will not store 2 of the same commands if they were entered twice in a row. Likewise, "commands" containing exclusively of blank spaces will not be stored as well.
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
<div markdown="block" class="alert alert-info">:information_source: CCACommander will not store 2 of the same commands if they were entered twice in a row. Likewise, commands containing exclusively of blank spaces will not be stored as well.
```

**Review by SelwynAng — CHANGES_REQUESTED** [review]
Some nits to address.

File: docs/2101UserGuide.md [hunk 13]
```diff
@@ -206,34 +236,37 @@ As a CCA head, keeping track of event details is a major responsibility. CCAComm
...
-    <figcaption>After executing the `createEvent` command</figcaption>
+    <figcaption>Fig 4: After executing the <code>createEvent</code> command</figcaption>
 </figure>
 
 <hr class="command-separator">
 
+<div style="page-break-after: always;"></div>
 #### Edit an Event : `editEvent`
 
 If you forgot to note any important details, or realise that you made a mistake in any field of the event created,
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
If you forget to note any important details, or realise that you made a mistake in any field of the event created,
```

File: docs/2101UserGuide.md [hunk 14]
```diff
@@ -206,34 +236,37 @@ As a CCA head, keeping track of event details is a major responsibility. CCAComm
...
 the `editEvent` command is here to help.
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
the `editEvent` command is here to help!
```

File: docs/2101UserGuide.md [hunk 15]
```diff
@@ -242,17 +275,19 @@ Here is an example command where we edit an event at **index** 1, changing the *
 
 Are the number of events piling up? To clear the clutter, you can delete any unwanted event from CCACommander with `deleteEvent`!
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
Are your events piling up? To clear the clutter, you can delete any unwanted event from CCACommander with `deleteEvent`!
```

File: docs/2101UserGuide.md [hunk 16]
```diff
@@ -242,17 +275,19 @@ Here is an example command where we edit an event at **index** 1, changing the *
...
 
 <div markdown="block" class="alert alert-info">:information_source: The `EVENT_INDEX` parameter refers to the index number shown in the **currently displayed** event list.
 </div>
 
 <hr class="feature-class-separator">
 
+<div style="page-break-after: always;"></div>
 ### Enrolment Commands
 
 Enrolments are a way for you to manage your members’ involvement in your events. The enrolment commands below are the bread-and-butter for your participation tracking needs:
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
Enrolments are a way for you to manage your members’ involvements in your events. The enrolment commands below are the bread-and-butter for your participation tracking needs:
```

File: docs/2101UserGuide.md [hunk 17]
```diff
@@ -265,50 +300,55 @@ Here’s how you can use it:<br>
...
 
-### View commands
+<div style="page-break-after: always;"></div>
+### View Commands
 In CCACommander, you can quickly change the view to fit your current use case, be it to check for the enrolment of an event, 
 searching for a member and more!
 
 #### View Events of Member : `viewMember`
 
 If you want to view the events in which a particular member is enrolled, you can do so via the `viewMember` command.
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
If you want to view the events in which a particular member is enrolled to, you can do so via the `viewMember` command.
```

File: docs/2101UserGuide.md [hunk 18]
```diff
@@ -265,50 +300,55 @@ Here’s how you can use it:<br>
...
 
-Here’s how you can use it:
+Here’s how you can use it:<br>
 `viewMember MEMBER_INDEX`
 
 Here is an example command where we want to view the events in which the **member** at index 3 in our member list is enrolled:
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
Here is an example command where we want to view the events in which the **member** at index 3 in our member list is enrolled to:
```

File: docs/2101UserGuide.md [hunk 19]
```diff
@@ -451,12 +495,13 @@ Worry not as you can use the `undo` command to recover your data!
 
 Forgot how to edit a specific enrolment or how to unenrol members? The `help` command will allow you to clarify and recall any features of the app by providing you a link to our friendly and informative user guide.
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
Forgot how to edit a specific enrolment or how to unenrol members? The `help` command will allow you to clarify and recall any features of CCACommander by providing you with a link to our friendly and informative user guide.
```

File: docs/2101UserGuide.md [hunk 20]
```diff
@@ -465,10 +510,11 @@ Here’s how you can use it: `help`
...
 If it’s time to call it a day for your CCA tasks, you can use the `exit` command to quit the app.
 
-Here’s how you can use it: `exit`
+Here’s how you can use it:<br>
+`exit`
 
-<div markdown="block" class="alert alert-info">:information_source: The exit command exists for your convenience. 
-It performs the same task as the cross in the top bar and the Exit button in the File submenu item
+<div markdown="block" class="alert alert-info">:information_source: The <code>exit</code> command exists for your convenience. 
+It performs the same task as the cross in the top bar and the Exit button in the File sub-menu item
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]:
```suggestion
It performs the same task as the cross in the top bar and the Exit button in the File sub-menu item.
```

File: docs/2101UserGuide.md [hunk 21]
```diff
@@ -532,15 +580,16 @@ Action | Format, Examples
...
 **View events of member** | `viewMember MEMBER_INDEX` <br> e.g.`viewMember 1`
-**Find member in list** | `findMember KEYWORD [MORE_KEYWORDS]` <br> e.g.`findMember alice`
-**Find event in list** | `findEvent KEYWORD [MORE_KEYWORDS]` <br> e.g.`findEvent party`
+**Find member in list** | `findMember MEMBER_NAME [MORE_NAMES]...` <br> e.g.`findMember alice`
+**Find event in list** | `findEvent EVENT_NAME [MORE_NAMES]...` <br> e.g.`findEvent party`
 **List all members and all events** | `list`
 **Redo** | `redo`
 **Undo** | `undo`
 **Help** | `help`
 **Exit** | `exit`
```
Comment (SelwynAng) [inline docs/2101UserGuide.md]: Commands in Command Summary are in different order from those in feature description.

**Review by SelwynAng — CHANGES_REQUESTED** [review]
Some nits to address.

File: docs/2101UserGuide.md (hunk 9 above)
Comment (andrechuakj) [reply docs/2101UserGuide.md]: Sure

File: docs/2101UserGuide.md (hunk 10 above)
Comment (andrechuakj) [reply docs/2101UserGuide.md]:
```suggestion
You can quickly recall previously entered commands for convenience as CCACommander stores almost all (see the info card below to see what commands are stored) commands entered by the user. 

```

File: docs/images/viewEventPostCommand.png
Comment (SelwynAng) [inline docs/images/viewEventPostCommand.png:1]: I think you mixed up the screenshots with `viewMember`

File: docs/images/viewMemberPostCommand.png
Comment (SelwynAng) [inline docs/images/viewMemberPostCommand.png:1]: I think you mixed up the screenshot with `viewEvent`

**Review by SelwynAng — CHANGES_REQUESTED** [review]
Some nits

**Review by SelwynAng — APPROVED** [review]
LGTMMMMMMMM

**Pull request merged**